smart-contracts/contracts/Watcher.sol

```diff
@@ -1,19 +1,20 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/access/AccessControl.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
 
 import "./interfaces/ISovrynSwapNetwork.sol";
 import "./interfaces/ISovrynProtocol.sol";
 import "./interfaces/IPriceFeeds.sol";
 import "./interfaces/IWRBTCToken.sol";
 
-contract Watcher is AccessControl {
-    using SafeERC20 for IERC20;
+contract Watcher is Initializable, AccessControlUpgradeable {
+    using SafeERC20Upgradeable for IERC20Upgradeable;
```

Comment:
If the Watcher has been already deployed as an upgradable proxy, then this change would break storage layout.
(I assume it is not the case)